go.mod

```diff
@@ -82,3 +84,5 @@ replace (
 // Supports go-ipld-prime v7
 // TODO: remove once https://github.com/filecoin-project/go-hamt-ipld/pull/70 is merged to github.com/filecoin-project/go-hamt-ipld
 replace github.com/filecoin-project/go-hamt-ipld/v2 => github.com/willscott/go-hamt-ipld/v2 v2.0.1-0.20210225034344-6d6dfa9b3960
+
+replace github.com/filecoin-project/lotus => ../lotus
```

Comment:
other half of this lives here: https://github.com/filecoin-project/lotus/pull/6389